tests/.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__
```

Comment:
Please uncommit this file and the pyc file

Reply:
OK